spec/paperclip/io_adapters/data_uri_adapter_spec.rb

```diff
@@ -80,4 +145,11 @@ def original_base64_content
   def original_file_contents
     @original_file_contents ||= File.read(fixture_file('5k.png'))
   end
+  
+  def svg_base64_content
+    Base64.encode64(svg_file_contents)
+  end  
+  def svg_file_contents
+    @svg_file_contents ||= File.read(fixture_file('good.svg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.